modules/theme/src/definitions/views/admin-portal.less

```diff
@@ -115,9 +115,7 @@
 }
 
 // VARIABLES
-@super-treeview-text-color: #000;
-@super-treeview-arrow-color: #000;
-@super-treeview-tree-stem-color: #000;
+@super-treeview-tree-stem-color: #f3f3f3;
 
 .super-treeview * {
```

Comment:
Shall we change this to just `treeview` ? may be you can make this a collection item instead

Reply:
fixed with commit 6cb1c41